src/tools/msc.lua

```diff
@@ -28,6 +28,15 @@
 -- Returns list of C compiler flags for a configuration.
 --
 
+	local function getruntimeflag(cfg, isstatic)
+		local rt = cfg.runtime
+		local flag = iif(isstatic, "/MT", "/MD")
+		if (rt == "Debug") or (rt == nil and config.isDebugBuild(cfg))  then
```

Comment:
Should `(rt == "Debug")` go in `config.isDebugBuild()`? Seems like this should be consistent behavior across tools.

Reply:
Depends on what the definition of `config.isDebugBuild()` is. You may have a configuration with debug symbols, no optimisations but using the runtime library. I would consider that a debug build. About the other way around, I can't tell.

Reply:
I am still unsure about this. The [description of `isDebugBuild`](https://github.com/premake/premake-core/blob/1aeb39dfcef8d5d9fdb893c95bd6d3c6ad150cb6/src/base/config.lua#L427-L431) hints it chooses the MS runtime version but as stated you may want to have symbols without using the Debug runtime. At least to me, the name of the function and what it does fit very well together.

Reply:
Let's leave `isDebugBuild()` alone for now, until there's a more compelling reason to change it.